src/AbstractDbEntityFetcher.php

```diff
@@ -75,19 +75,19 @@ protected function getLimitSql($limit, array $pageItem = [])
     /**
      * Helper method to get pagination result array if pagination is requested.
      *
-     * @param array $objects
+     * @param array $rows
      * @param bool  $pagination
      * @return array
      */
-    protected function getFetchPaginationResult(array $objects, $pagination)
+    protected function getFetchPaginationResult(array $rows, $pagination)
```

Comment:
A name like `getDbEntitiesFromRowsPaginated` would be more in line with the new behavior and the existing `getDbEntitiesFromRows` method.